# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/screen/default-histsize.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- screen-5.0.1/screen.h.vanilla	2025-06-17 14:20:34.538366199 +0200
+++ screen-5.0.1/screen.h	2025-06-17 14:21:31.143172472 +0200
@@ -51,8 +51,8 @@
  * if a nasty user really wants to try a history of 3000 lines on all 10
  * windows, he will allocate 8 MegaBytes of memory, which is quite enough.
  */
-#define MAXHISTHEIGHT		3000
-#define DEFAULTHISTHEIGHT	100
+#define MAXHISTHEIGHT		10000
+#define DEFAULTHISTHEIGHT	1000
 #define DEFAULT_BUFFERFILE	"/tmp/screen-exchange"
 
 struct mode {
